### ISSUE TITLE
Debug.MIDebugLaunch doesn't stop at main

### ISSUE THREAD

**Issue by Spongman** [Microsoft/MIEngine#539]
It's not stopping on main. and it doesn't hit any of my breakpoints.

It stops when an exception is thrown, or if I call Debugger.Break(), though.

I'm using the version included in the docker tools, and i have the latest clrdbg.

**Comment by gregg-miskelly** [comment]
@Spongman is this the same issue as #540?

**Comment by Spongman** [comment]
no, it was masked by #540, but once I built the dll (and pdbs) on linux, then the breakpoints started working. it still doesn't stop at main, though.

**Comment by gregg-miskelly** [comment]
Correct, Debug.MIDebugExec currently doesn't accept a '/StopAtEntryPoint' switch. It is hard coded to always continue from it.
